spec/defines/server/role_spec.rb

```diff
@@ -50,18 +50,20 @@
 
     it 'has create role for "test" user with password as ****' do
       expect(subject).to contain_postgresql_psql('CREATE ROLE test ENCRYPTED PASSWORD ****')
-        .with('command' => 'Sensitive [value redacted]',
-              'sensitive' => 'true',
-              'unless' => "SELECT 1 FROM pg_roles WHERE rolname = 'test'",
-              'port' => '5432')
+        .with('command'     => sensitive(%(CREATE ROLE "test" ENCRYPTED PASSWORD 'new-pa$s' LOGIN NOCREATEROLE NOCREATEDB NOSUPERUSER  CONNECTION LIMIT -1)),
+              'sensitive'   => 'true',
+              'unless'      => "SELECT 1 FROM pg_roles WHERE rolname = 'test'",
+              'port'        => '5432')
     end
 
     it 'has alter role for "test" user with password as ****' do
       expect(subject).to contain_postgresql_psql('ALTER ROLE test ENCRYPTED PASSWORD ****')
-        .with('command' => 'Sensitive [value redacted]',
-              'sensitive' => 'true',
-              'unless' => 'Sensitive [value redacted]',
-              'port' => '5432')
+        .with('command'     => sensitive(%(ALTER ROLE "test" ENCRYPTED PASSWORD 'Sensitive [value redacted]')),
+              # FIXME: This is obviously wrong                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
+              'sensitive'   => 'true',
+              'unless'      => sensitive(%(SELECT 1 FROM pg_shadow WHERE usename = 'test' AND passwd = 'Sensitive [value redacted]')),
+              # FIXME: This is obviously wrong                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
+              'port'        => '5432')
```

Comment:
Isn't this PR about fixing these two FIXME :wink:?

I rebased #1405 on top of the main branch (it should fix the `Layout/HashAlignment` issues we see above), can you rebase your first commit on top of my branch and adjust the test suite which should now work as expected, and force push so that the PR is updated?

Reply:
You are absolutely right! The result of a rebase without fully reviewing every single line. I've rebased again and fixed the tests.